src/main/java/com/kryptnostic/rhizome/cassandra/CassandraTableBuilder.java

```diff
@@ -267,6 +267,24 @@ public Where buildLoadQuery() {
     public Where buildLoadByPartitionKeyQuery() {
         return buildLoadQuery( partitionKeyColumnDefs() );
     }
+    
+    public Where buildLoadByPartitionKeyQueryWithStaticColumns() {
```

Comment:
Suggested fix here is make `allColumns(...)` include static columns (which is correct behavior).